graph-db/src/utils/cmr/getEchoToken.js

```diff
@@ -0,0 +1,22 @@
+const { getSecureParam } = require('../getSecureParam');
+
+/**
+ * getEchoToken: Fetch token for CMR requests
+ * @returns {String} ECHO Token for CMR requests
+ * @throws {Error} If no token is found. CMR will not return anything
+ * if no token is supplied.
+ */
+ exports.getEchoToken = async () => {
+    console.log(process.env.CMR_ENVIRONMENT);
+    const response = getSecureParam(
+      `/${process.env.CMR_ENVIRONMENT}/browse-scaler/CMR_ECHO_SYSTEM_TOKEN`
+    );
+  
+    if (response === undefined) {
```

Comment:
Could this ever be `null` or an empty string instead of undefined? May want to simplify to `if (!response)`